### ISSUE TITLE
Takes a lot of time to find the best model.

### ISSUE THREAD

**Issue by K0MPLEXWorksTogether** [K0MPLEXWorksTogether/DataWhisperer#1]
https://www.youtube.com/watch?v=lo0_Wnk3mqM&feature=youtu.be
As you can see, the time taken to find the best model is a lot.

A fix to this is much appreciated.

**Comment by srirammandalika** [comment]
For my clarification, as in the YT video "Finding the best models for the dataset...." means that an appropriate model to work with is being searched + Training?, if I understood it correctly:)

**Comment by K0MPLEXWorksTogether** [comment]
Yeah, thats the exact problem I am looking to solve. It takes a huge amount of time to do this. @srirammandalika

**Comment by Ujjwal29** [comment]
Considering the tool's applicability to diverse data, model selection can be complex and time-consuming. We can streamline this process by starting with a suitable model and implementing a performance threshold. Once achieved, the selection can be finalized.

**Comment by K0MPLEXWorksTogether** [comment]
Cool. I'll take you up on that.
